widgetastic_manageiq/__init__.py

```diff
@@ -2651,12 +2651,15 @@ class ListView(EntitiesConditionalView):
 
         @property
         def entity_names(self):
-            """ looks for entities and extracts their names
+            """ looks for entities and extracts their names or keys
```

Comment:
**REQUIRED:** I believe @izapolsk is introducing a search for entities by multiple col types....so this change shouldn't be necessary, can you confirm @izapolsk

Reply:
@psav, yes this is true. I'm working on it. ETA - next week.

Reply:
@izapolsk, for now, can I go with this small change or wait?